### ISSUE TITLE
There's some reason why I have to pass `redraw: true` when dispatch actions?

### ISSUE THREAD

**Issue by rkmax** [colinbate/mithril-redux#5]
I think the 'redraw' is the expected behavior

**Comment by colinbate** [comment]
Yes, it should be in theory. Originally I did this so that it could co-exist with the `m.mount()` feature of Mithril which does auto-redrawing. I didn't want to redraw more often than necessary. However, the appropriate thing would be to have the rendering controlled based on a subscription to the Redux store.

A basic implementation shouldn't be hard, but it may need some optimizations that I'm not aware of.

I'll take a look at this when I can, but it might be a couple weeks before I can get anything done.
